saml-idp.py

```diff
@@ -228,7 +252,8 @@ def root():
     relay_state = request.values.get('RelayState')
     return f'''<HTML><BODY><CENTER><H1>Login - Step 1</H1>
 <FORM ACTION="https://{HOST}:{PORT}/login" METHOD=POST>
-Username: <INPUT TYPE=TEXT VALUE="admin" NAME="username"><BR>
+Service Provider Metadata Entity ID +/-: <INPUT TYPE=TEXT NAME="sp_entity_id" VALUE="https://www.okta.com/saml2/service-provider/spkrktcxhbfmkgfelqsv"><BR>
```

Comment:
Is this a generic okta or specific to Sitka Okta?

Reply:
Specific to an Idp listed in okta settings, this one being for the new "local dev" one I added.  But I see where you are going with this, I should remove the default value so its entered each time :)

Reply:
You did read my mind! And if you leave it, just make it clear in the documentation or a comment what it refers to. :D